docs/source/contributing.rst

```diff
@@ -82,59 +77,37 @@ Run the units tests to see if all is fine::
 yourself in the AUTHORS file with your email address.
 
-If the trunk has been updated while you were hacking, you should update your
-local trunk branch, and merge modification in **your** branch::
-
-    $ cd ../trunk
-    $ bzr pull trunk
-    $ cd ../cool-new-feature
-    $ bzr merge ../trunk
-
-If you have conflicts, you must solve them. Refer to `conflicts guide`_.
+If the master has been updated while you were hacking, you should update your
+local master branch, and merge modification in **your** branch::
 
-.. _`conflicts guide`: http://doc.bazaar.canonical.com/bzr.0.92/en/user-guide/conflicts.html
+    $ git checkout master
+    $ git pull origin master
+    $ git checkout cool-new-feature
+    $ git merge master
 
-Once you don't have any conflict anymore, you must commit the changes related
-to the merge. Use a clear commit message, like::
 
-    Updating branch by merging the last trunk version.
+When you have done some changes or solved a bug, add and commit the changes.
+Afterwards, you need to push your work to your own fork on GitHub (where cool-new-feature
+is the name of your local branch which you changed.)::
 
-Pushing your work to your own branch on Launchpad (where *ploum* is your
-Launchpad username)::
+    $ git push origin cool-new-feature
 
-    $ bzr push lp:~ploum/gtg/cool-new-feature
+If you have made changes and pushed them to your forked master branch on GitHub,
+you can do a pull request to merge your work with the original GTG master.
+To do this, go to your account on GitHub and click on "New Pull Request".
 
-Alternatively, if you want other gtg users to be able to write to your branch,
-push it in the gtg-user group (you have to be part of it)::
-
-    $ bzr push lp:~gtg-user/gtg/ploum_branch
-
-Ask for a merge request and comment on the corresponding bug. (Open one if
-there is none). Add the tag *toreview* to the bug in Launchpad. This is very
+Create a pull request and comment on the corresponding bug. (Open one if
+there is none). Add the tag *toreview* to the bug in GitHub. This is very
 important and ensures we are not letting a patch rotting.
 
-You can file a bug at https://bugs.launchpad.net/gtg/+filebug.
-
-To ask for a merge request, run::
-
-$ cd cool-new-feature
-$ bzr lp-open
-
-This will open the branch's web page on Launchpad. From there, click *Propose for merging*.
+You can file a bug at https://github.com/getting-things-gnome/gtg/issues/new
 
 If your branch is solving specific reported bugs, please also register your
-branch to these bugs (there is an link for that in each bug report page). It
-allows to link together all related resources (which in turn is useful to dig
-out precious information from all the discussions that happened around those
-bugs).
+branch to these bugs. It allows to link together all related resources (which 
+in turn is useful to dig out precious information from all the discussions that 
+happened around those bugs).
```

Comment:
Mentioning the number of bug is enough: https://github.com/blog/957-introducing-issue-mentions